src/node/mini_miner.h

```diff
@@ -138,7 +138,7 @@ class MiniMiner
     MiniMiner(const CTxMemPool& mempool, const std::vector<COutPoint>& outpoints);
 
     /** Constructor in which the MiniMinerMempoolEntry entries have been constructed manually,
```

Comment:
Nit: First sentence ends in a comma now.

Reply:
fixed in [b4b01d3](https://github.com/bitcoin/bitcoin/pull/28808/commits/b4b01d3fb42e7b688d97b75f57cfe18cfca6d943)